src/Bicep.Core/TypeSystem/Azrm/AzrmResourceTypeProvider.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Bicep.Core.Extensions;
+using Bicep.Core.Resources;
+
+namespace Bicep.Core.TypeSystem.Azrm
```

Comment:
I initially used `Arm` but switched to `Azrm` because `Arm` is in our `.gitignore` file. It may be possible to solve that if you think it's important to use `Arm` :)

Reply:
What we used to call `azrm` I think we're calling `az` now.

Reply:
Good point. Az works.